code/datums/spells/lichdom.dm

```diff
@@ -104,7 +104,7 @@
 
 			if(!marked_item)
 				M << "<span class='caution'>You must hold an item you wish to make your phylactery...</span>"
-
+				return
 			spawn(50)
```

Comment:
split everything after spawn into 'newproc'
then `addtimer(src, "newproc", 50, FALSE, M)`
just ask coderbus if you're confused